datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -503,12 +503,13 @@ private void validateMediumDutySdxEnablement(SdxClusterRequest sdxClusterRequest
         ValidationResultBuilder validationBuilder = new ValidationResultBuilder();
         if (SdxClusterShape.MEDIUM_DUTY_HA.equals(sdxClusterRequest.getClusterShape())) {
             boolean mediumDutySdxEntitlementEnabled = entitlementService.mediumDutySdxEnabled(Crn.safeFromString(environment.getCreator()).getAccountId());
-            if (!mediumDutySdxEntitlementEnabled) {
+            CloudPlatform cloudPlatform = EnumUtils.getEnumIgnoreCase(CloudPlatform.class, environment.getCloudPlatform());
+            if (!AWS.equals(cloudPlatform) && !mediumDutySdxEntitlementEnabled) {
                 validationBuilder.error("Provisioning a medium duty data lake cluster is not enabled for this account. " +
                         "Contact Cloudera support to enable CDP_MEDIUM_DUTY_SDX entitlement for the account.");
             }
             if (!isMediumDutySdxSupported(sdxClusterRequest.getRuntime())) {
-                validationBuilder.error("Provisioning a Medium Duty SDX shape is only valid for CM version >= 7.2.2 and not " + sdxClusterRequest.getRuntime());
```

Comment:
**suggestion:** You _might_ consider extracting the message String to a `public static final String` so that it's a bit easier to validate test messages -- you don't have to copy/paste or synchronize them in the future!

Reply:
I almost want to say !(AWS or entitled) is easier to read?